modules/create-resource-init-bundle.adoc

```diff
@@ -56,14 +56,16 @@ If you are installing by using Helm charts, do not perform this step. Complete t
 
 .Prerequisites
 * You must have generated an init bundle containing secrets.
+* You must have created the `stackrox` project, or namespace, on the cluster where secured cluster services will be installed. Using `stackrox` for the project is not required, but ensures that vulnerabilities for {product-title-short} processes are not reported when scanning your clusters.
 
 .Procedure
+//Show for OpenShift
 ifdef::openshift[]
-To create resources, perform one of the following steps:
+To create resources, perform only one of the following steps:
 
-* In the {ocp} web console, in the top menu, click *+* to open the *Import YAML* page. You can drag the init bundle file or copy and paste its contents into the editor, and then click *Create*.
+* Create resources using the {ocp} web console: In the {ocp} web console, make sure that you are in the `stackrox` namespace. In the top menu, click *+* to open the *Import YAML* page. You can drag the init bundle file or copy and paste its contents into the editor, and then click *Create*. If you successfully completed this step, the display shows that the `collector-tls`, `sensor-tls`, and admission-control-tls` resources were created.
```

Comment:
To me, _If you successfully completed this step_ sounds like a challenge.
```suggestion
* Create resources using the {ocp} web console: In the {ocp} web console, make sure that you are in the `stackrox` namespace. In the top menu, click *+* to open the *Import YAML* page. You can drag the init bundle file or copy and paste its contents into the editor, and then click *Create*. When the command is complete, the display shows that the `collector-tls`, `sensor-tls`, and admission-control-tls` resources were created.
```

Reply:
It was a challenge for me :D but I'll reword....